### DISCUSSION TITLE
PKAddPassButton

### DISCUSSION THREAD

**Discussion by liapisn** [alexandercerutti/passkit-generator#107] [General]
Is there any way that we can add button at the left down/right down corner ?

**Comment by alexandercerutti** [comment]
Hey there @liapisn, which corner are you talking about? Do you mind to add more contextual details so I can understand better, please?

**Reply by liapisn** [reply]
I want to add a button like in the photo bellow at the down left corner which will navigate to a site 
![Screenshot from 2022-03-09 13-20-07](https://user-images.githubusercontent.com/57102404/157431962-f926e18c-49a3-421f-a489-a7416962b1eb.png)

**Reply by alexandercerutti** [reply]
The only item I know you can set and see on the bottom left corner of a pass, is the icon of the application you can link throught pass.json property `associatedStoreIdentifiers`.

The icon should be the the one you add to your bundle.

It opens the application, if downloaded, or the App store to download it, when it is tapped.

I'm not aware or another way to set a link to a website which isn't `attributedValue` on a field.

Hope this helps 😄 

P.s. this is way different from a PKAddPassButton. That button allows iOS Apps to request for a Pass and show it

**Reply by liapisn** [reply]
Ok thank you very much for your help and your fast resonse
